tools/test_deploy.py

```diff
@@ -0,0 +1,156 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# Copy from https://github.com/open-mmlab/mmdeploy/blob/master/tools/test.py
```

Comment:
do not think it's a good idea to copy this file.

Reply:
If we do not copy this file, we need to jump into mmdeploy path and run it for deploying. It will be a little matter.